shlr/zip/zip/zipint.h

```diff
@@ -46,6 +46,10 @@
 #include <io.h>
 #endif
 
+#ifdef _MSC_VER
+typedef long ssize_t;
```

Comment:
Don define this, go to /zip/config.h and add this condition to line #define HAVE_SSIZE_T 1